airflow/models.py

```diff
@@ -790,6 +790,10 @@ class TaskInstance(Base, LoggingMixin):
         Index('ti_pool', pool, state, priority_weight),
     )
 
+    # Is this TaskInstance being currently running within `airflow run --raw`.
+    # Not persisted to the database so only valid for the current process
+    is_raw = False
```

Comment:
Make this part of and default to fAlse. Then use init_run_context to set it

Reply:
Of init I meant

Reply:
oh god yes, much nicer. Not sure what I was thinking